test-capitalizations/verify-capitalization-helpfile-qml-files.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+debug="false"
+debug_printf() 
+{
+	if [ ! ${debug} == "false" ]; then
+		printf "$@"
+	fi
+}
+
+ROOT=${1-.}
+if [ "${ROOT}" == '.' ]; then
+	ROOT=$(pwd)
+fi
+
+DESCRIPTION_QML="${ROOT}/inst/Description.qml"
+NAMESPACE="${ROOT}/NAMESPACE"
+HELP_FILES_DIR="${ROOT}/inst/help"
+QML_FILES_DIR="${ROOT}/inst/qml"
+
+debug_printf "root = %s\n" $ROOT
+debug_printf "DESCRIPTION_QML = %s\n" $DESCRIPTION_QML
+
+# FUNCTIONS=$(sed -n 's/\s*func\s*:\s*\"\(.*\)\"/\1/p' "${DESCRIPTION_QML}")
+FUNCTIONS=$(sed -n 's/.*func\s*:\s*\"\(.*\)\".*/\1/p' "${DESCRIPTION_QML}")
```

Comment:
I'm not exactly sure, but this does not work on my Mac. I know that `sed` that ships with macOS is a bit different, and that could be the cause. However, if I remove the spaces between `func`, `:`, `FunctionName` and have `func:FunctionName` then your script picks it up. My guess is that it cannot read tab character, but I couldn't convince it to do it by adding `\s*|\t*`.

Reply:
>this does not work on my Mac

It's not intended to be Linux only. I'll look into this so that it works on any Unix platform.

FWIW, with the master of jaspCircular, I get
```
jasp-actions/test-capitalizations/verify-capitalization-helpfile-qml-files.sh jaspCircular 
found functions:
CircularStatisticsDescriptives
CircularStatisticsOneSampleTests
CircularStatisticsMultipleSampleTests
These helpfiles have incorrect capitalization:
        circularstatisticsdescriptives.md
        circularstatisticsonesampletests.md
        circularstatisticsmultiplesampletests.md

EXITCODE = 2
```

Reply:
I know, it’s fine if it only runs on Linux but I could never be sure with RegEx so I thought I would mention it anyway.